src/map/status.cpp

```diff
@@ -558,7 +558,7 @@ efst_type StatusDatabase::getIcon(sc_type type) {
  * @param type: SC type
  * @return cal_flag: Calc value 
  **/
-uint64 StatusDatabase::getCalcFlag(sc_type type) {
+std::bitset<SCB_MAX> StatusDatabase::getCalcFlag(sc_type type) {
 	std::shared_ptr<s_status_change_db> status = status_db.find(type);
 
 	return status ? status->calc_flag : SCB_NONE;
```

Comment:
Returning SCB_NONE here looks strange?
I think this should be an empty bitset now?